_posts/1-2-27-Lightbox.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: doc
+permalink: /docs/ui-toolkit/components/lightbox
+title: Lightbox
+section: UI toolkit
+---
+
+# Lightbox
+
+This Lightbox is Shoutem's fork of [`react-native-lightbox`](https://github.com/oblador/react-native-lightbox) component.  
+All content within Lightbox is transformable, meaning it can be pinched to zoom, panned (translated) etc. 
+
+## Lightbox
+![alt text](https://cloud.githubusercontent.com/assets/378279/9074360/16eac5d6-3b09-11e5-90af-a69980e9f4be.gif "Lightbox"){:.docs-component-image}
+
+#### JSX Declaration
+```JSX
+<Lightbox>
+  <Image
+    style={% raw %}{{ height: 300 }}{% endraw %}
+    source={% raw %}{{ uri: 'http://knittingisawesome.com/wp-content/uploads/2012/12/cat-wearing-a-reindeer-hat1.jpg' }}{% endraw %}
+  />
+</Lightbox>
+```
+
+#### Props
+
+* **activeProps** : object  
+  - Optional set of props applied to the content component when in lightbox mode. Usable for applying custom styles or higher resolution image source.
+* **renderHeader(close)** : function
+  - Callback function that should return custom header instead of default with X button in upper left corner. 
+* **renderContent** : function
+  - Callback function that should return custom Lightbox content instead of default child content.
+* **onClose**: function
+  - Callback function triggered when Lightbox is closed.
+* **onOpen**: function
+  - Callback function triggered when Lightbox is opened.
```

Comment:
removed `callback function` where it wasn't necessary and rephrased a bit. also marked components. should be good now.